### DISCUSSION TITLE
Awesome work!

### DISCUSSION THREAD

**Discussion by INDIGI-CO-UK** [pguyot/Einstein#153] [Show and tell]
I want to thank you all so much for your hard work and commitment to this project! This is really great that you are still developing this emulator and we all can use it! I use it every day when I'm working on my computer. Now I'm waiting for my first Android smartphone and I can't wait to use Einstein on it :)

Thank you very very much!

Cheers,
Pawel

**Comment by MatthiasWM** [comment]
Thanks Pawel. There has been a ton of movement in the code in the last weeks thanks to your online conference. Are we still up for the meeting tomorrow. I was making sure to get the beat 4.17 out today, so we can announce it tomorrow and get suggestions for 2022.5.0.

**Reply by INDIGI-CO-UK** [reply]
Yes, of course, that we are! I will send reminders on NewtonTalk and other media about it! Can't wait to see your demo! :D

**Reply by MaxDamage9** [reply]
I wonder if there is a recommended platform these days for running Einstein? 

I am an OLDTIME Newton user (Hi y'all!) and wonder if there is a particular phone or tablet that works best or can be booted directly to Einstein?

Thanks to all of you!
Marty (stillnewt.org)
